recipe/meta.yaml

```diff
@@ -17,6 +17,8 @@ requirements:
   build:
     - python
     - pip
+    - wheel >=0.31.0
+    - twine >=1.11.0
```

Comment:
@lsetiawan these are listed as [setup_requires](https://github.com/Freshwater-Initiative/Observatory/blob/master/setup.py#L49) so they must be build dependencies.

Although, one might argue that those should not be listed there at all since they are actually dev dependencies and not really needed for `setup`.

Reply:
Ah I see. Good to know! Thank you! :smile: